Monika After Story/game/script-topics.rpy

```diff
@@ -15751,3 +15751,130 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_impermanence",
+            category=["philosophy"],
+            random=True,
+            prompt="Impermanence",
+            sensitive=True
+        )
+    )
+
+label monika_impermanence:
+    $ wants_full_topic = None
+    if not mas_getEVL_shown_count("monika_impermanence"):
+        # first time intro from random
+        m 2ruc "..."
+        m 2euc "You know...{w=0.5}there's something that's been on my mind lately."
+
+    else:
+        # repeat from menu intro
+        # safe to assume the player wants to hear what Moni has to say, skip the main choice from here
+        m 2euc "You know, I find myself thinking about some dark stuff occasionally."
+        $ wants_full_topic = True
+
+    m 1eud "Concepts like impermanence..." # manual extends
+    extend 1ekc " Nihilism..."
+    extend 1ekd " Depression."
+
+    if mas_isMoniAff(higher=True):
+        m 3eksdla "I'm not going through anything like that myself, of course."
+        m 1tsa "Courtesy of a certain someone."
+    else:
+        m 1wud "Ah-{w=0.2}{nw}"
+        extend 1eub " I don't mean to worry you, I'm not going through it myself, or anything like that."
```

Comment:
```suggestion
        extend 1eub " I don't mean to worry you, I'm not going through depression myself, or anything like that."
```
I know the isMoniAff path was added after one of my previous comment, but thinking more about it, I think it could be kept to this line only if changed like so.
The thing is, some of her topics show she'd be concerned about nihilism and and impermanence (`monika_eternity`, `monika_nihilism`,`monika_secret`'s base 64 text).

Reply:
So axe the conditional entirely and just keep the altered else branch?